bundles/nl.esi.comma.project.standard.cli/dist/start-simulator.bat

```diff
@@ -1,14 +1,21 @@
 @ECHO OFF
 SETLOCAL
 
+echo Using python environment: %BPMN4S_PYTHON%
+
+IF not defined BPMN4S_PYTHON (
```

Comment:
Please make the usage of this environment variable optional. If it is not set, just default to `python.exe`. We only want to offer the possibility to choose the python installation, we do not want to make things more difficult if not needed.